src/adserver.js

```diff
@@ -0,0 +1,57 @@
+import {formatQS} from './url';
+
+//Adserver parent class
+var adserver = function(attr) {
+  var that = {};
+  that.name = attr.adserver;
+  that.code = attr.code;
+  that.getWinningBidByCode = function() {
+    var bidObject = $$PREBID_GLOBAL$$._bidsReceived.find(bid => bid.adUnitCode === that.code);
+    return bidObject;
+  };
+  return that;
+};
+
+//DFP ad server
+exports.dfpAdserver = function (options, urlComponents) {
```

Comment:
usually our exports are `verbs` and not `nouns`. Although this is more style, it does help with consistency in the project.